imod/protocols/protocol_fiducialAlignment.py

```diff
@@ -136,14 +199,19 @@ def _defineParams(self, form):
                       default=3,
                       label='Rotation solution type',
                       display=params.EnumParam.DISPLAY_HLIST,
-                      help='Type of rotation solution.')
+                      help='Type of rotation solution: See rotOption in tiltalign IMOD command \n'
```

Comment:
Would it be better looking if we changed all params displayed here to DISPLAY_COMBO?

Reply:
I had the same question in this protocol, hehehe so I took a few users of tomo users of our center and the facility. The answer of unanimous, they prefer this choice display instead of the combo. The reason: it looks more close to IMOD, and they prefer to see all options. 
Them, I asked the same for tilt series normalization and tomogram normalization. In those protocols hardly ever they change the choice, so the display_combo was ok.